graylog2-web-interface/packages/stylelint-config-graylog/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "stylelint-config-graylog",
+  "version": "1.0.0",
+  "description": "Graylog StyleLint config for web interface plugin authors",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "graylog",
+    "stylelint"
+  ],
+  "author": "Graylog, Inc. <hello@graylog.com>",
+  "license": "SSPL-1.0",
+  "dependencies": {
+    "stylelint": "13.12.0",
```

Comment:
Isn't stylelint a peer dependency of the config? It's needed for the main project, as well as this config, but if we'd change the config, shouldn't stylelint still be in the main project?